gui/elements/_viewer.py

```diff
@@ -274,36 +159,30 @@ def _set_annotation_mode(self, bool):
                 self._qt.canvas.native.setCursor(Qt.CrossCursor)
             else:
                 self._qt.canvas.native.setCursor(self._disabled_cursor)
-            self._status_widget.show()
+            self._help = 'hold <space> to pan/zoom'
         else:
             self.annotation = False
             self._qt.view.interactive = True
             self._qt.canvas.native.setCursor(QCursor())
-            self._status_widget.hide()
+            self._help = ''
+        self.helpChanged.emit(self._help)
         self._update_statusBar()
 
-    def _update_index(self, event):
-        visual = self.layers[0]._node
-        tr = self._canvas.scene.node_transform(self.layers[0]._node)
-        pos = tr.map(event.pos)
-        pos = [clip(pos[1],0,self.max_shape[0]-1), clip(pos[0],0,self.max_shape[1]-1)]
-        self._index = copy(self.indices)
-        self._index[0] = int(pos[0])
-        self._index[1] = int(pos[1])
-
     def _update_active_layers(self):
         from ..layers._image_layer import Image
         from ..layers._markers_layer import Markers
-
         top_markers = []
         for i, layer in enumerate(self.layers[::-1]):
             if layer.visible and isinstance(layer, Image):
                 top_image = len(self.layers) - 1 - i
                 break
             elif layer.visible and isinstance(layer, Markers):
-                top_markers.append(len(self.layers) - 1 - i)
-                coord = [self._index[1],self._index[0],*self._index[2:]]
-                layer._set_selected_markers(coord)
+                if self.dimensions._index is None:
+                    pass
+                else:
+                    top_markers.append(len(self.layers) - 1 - i)
+                    coord = [self.dimensions._index[1],self.dimensions._index[0],*self.dimensions._index[2:]]
```

Comment:
This line clearly demonstrates that we are doing something wrong with display dimensions.

Reply:
Seriously.

Reply:
Yup, agreed - the markers are expecting the first two coordinates in the reversed order! This is on the radar for fixing in #86